src/Car.php

```diff
@@ -2,20 +2,30 @@
 
 namespace CarStore;
 
+require_once 'vendor/autoload.php';
+require_once 'src/utils.php';
+
+use CarStore\CarModel;
+
+
 readonly class Car
 {
     public string $name;
     public int $year_made;
     public float $zero_sixty;
     public float $price;
     public string $brand;
+    public int $id;
```

Comment:
You need to make the property types nullable as well to match the constructor params